test/jit/test_freezing.py

```diff
@@ -1563,6 +1563,48 @@ def to_dense(inp):
                     self.assertEqual(out_eager.is_mkldnn, out_scripted.is_mkldnn)
                     self.assertEqual(to_dense(inp), to_dense(inp))
 
+    @unittest.skipIf(not torch._C.has_mkldnn, "MKL-DNN build is disabled")
+    def test_linear_to_mkldnn(self):
+        # use until github.com/pytorch/pytorch/pull/50856 land
+        class LinearMod(nn.Linear):
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+
+            def forward(self, input):
+                return torch._C._nn.linear(input, self.weight, self.bias)
```

Comment:
this is to get rid of an if?

Reply:
This is to compile it as one op. This is how it's done on master as of now but wasnt landed yet when i submitted this PR.